cluster-test.sh

```diff
@@ -69,7 +69,7 @@ check_metrics() {
     sleep 1
     metrics_got=$(curl -s localhost:9090/metrics | grep "^transflect_" | grep --invert-match "^transflect_ignored_total" | sort)
 
-    metrics_want='transflect_envoyfilters 2
+    metrics_want='transflect_envoyfilters 1
```

Comment:
oups, this was wrong all along.
the integration tests at this stage have:
created 2 filters, updated 1, and deleted 1 => 1 filters is still active.